docs/pages/components/Button.mdx

```diff
@@ -78,6 +78,30 @@ When you need a semantic button with the visual appearance of a link.
 <Button variant="tag">Tag</Button>
 ```
 
+### Badge
```

Comment:
Is there a description we have for when to use this pattern?

Reply:
Not that I'm aware of. Perhaps either @scurker or @ezirlingerDQ have a suggestion as to what we could put here?

Reply:
We don't have strict guidelines for descriptive documentation right now. Minimally, the pattern needs to have a documented example but not everything always has a description. Since the description for "tag" is also omitted, I'm not 100% sure yet what we would want to put here.